tox.ini

```diff
@@ -35,10 +55,14 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
+  {[testenv:build-dists]deps}
   collective.checkdocs
   twine
 usedevelop = False
+skip_install = true
 # Ref: https://twitter.com/di_codes/status/1044358639081975813
 commands =
-  python -m setup checkdocs check --metadata --restructuredtext --strict --verbose
-  twine check .tox/dist/*
+  {[testenv:build-dists]commands}
+  twine check {toxinidir}/dist/*
+whitelist_externals =
+    rm
```

Comment:
This is unnecessary

Reply:
@webknjaz This IS necessary because the inhered commands contain rm, and tox with throw a warning. If you remember few months back you were against setting a single whitelist_externals on molecule, so I suspect you didn't change your preference.

Reply:
We shouldn't inherit the commands in the first place. This env should remain decoupled and we need to prefer composition over inheritance in this case.